packages/purpose-process/src/services/validators.ts

```diff
@@ -81,12 +85,12 @@ export const assertConsistentFreeOfCharge = (
   }
 };
 
-export const assertOrganizationIsAConsumer = (
-  organizationId: TenantId,
-  consumerId: TenantId
+const assertOrganizationIsAConsumer = (
```

Comment:
Rename it to requester instead of organization, like for the producer assert and other process services 

```suggestion
const assertRequesterIsConsumer = (
```